libopeniscsiusr/Makefile

```diff
@@ -108,7 +108,7 @@ docs/man/$(EXTRA_MAN_FILES).gz: $(HEADERS)
 	    perl docs/doc-preclean.pl > "$(TEMPFILE)"
 	perl docs/kernel-doc -man "$(TEMPFILE)" | \
 	    perl docs/split-man.pl docs/man
-	-rm -f "$(TEMPFILE)"
+	-$(RM) -f "$(TEMPFILE)"
```

Comment:
The RM Make macro is already set to "rm -f" by default, so you need not keep the "-f". Please fix this here and in other Makefiles.

Reply:
Done, thanks!